app/controllers/application_controller/ci_processing.rb

```diff
@@ -1609,42 +1607,55 @@ def scanclusters
     cluster_button_operation('scan', 'Analysis')
   end
 
+  def each_host(hosts, task_name)
+    Host.where(:id => hosts).order("lower(name)").each do |host|
+      begin
+        yield host
+      rescue StandardError => bang
```

Comment:
Since StandardError is the default, we don't need to specify.  Also, I personally can't stand `bang` as a var name, and would prefer `err`, but that one is up to you :smile:.  So, `rescue => err`